pam/integration-tests/testdata/TestNativeAuthenticate/golden/authenticate_user_switching_auth_mode

```diff
@@ -3108,7 +3108,7 @@ Select action: r
 7 - Use a QR code
 8 - Authentication code
 Select authentication mode: invalid-selection
-Provided input can't be parsed as integer value
+PAM Error Message: Provided input can't be parsed as integer value
```

Comment:
I wonder if only `PAM Error:` is already enough though, having the `... Message:` feels a bit redundant. WDYT?

Reply:
Well, I didn't want to confuse those with actual PAM errors at the time, since those are messages sent by PAM with the "error type", which is different from a returned PAM error (string).